src/views/Public/Pages/Records/Edit.vue

```diff
@@ -15,7 +15,7 @@ export default {
 
   methods: {
     handleBack () {
-      this.$router.push({ name: 'page.record' })
+      this.$router.push({ name: 'page.record', params: { recordID: this.recordID, pageID: this.page.pageID } })
```

Comment:
Protip, `this.$router.push({ name: 'page.record' })` will be enough since params should be kept